src/webapp/pages/form-page/disease-outbreak-event/utils/mapEntityToInitialFormState.ts

```diff
@@ -0,0 +1,655 @@
+    suspectedDiseaseCode: "suspectedDiseaseCode",
+    notificationSourceCode: "notificationSourceCode",
+    areasAffectedProvinceIds: "areasAffectedProvinceIds",
+    areasAffectedDistrictIds: "areasAffectedDistrictIds",
+    incidentStatus: "incidentStatus",
+    emergedDate: "emergedDate",
+    emergedNarrative: "emergedNarrative",
+    detectedDate: "detectedDate",
+    detectedNarrative: "detectedNarrative",
+    notifiedDate: "notifiedDate",
+    notifiedNarrative: "notifiedNarrative",
+    initiateInvestigation: "initiateInvestigation",
+    conductEpidemiologicalAnalysis: "conductEpidemiologicalAnalysis",
+    laboratoryConfirmationDate: "laboratoryConfirmationDate",
+    laboratoryConfirmationNA: "laboratoryConfirmationNA",
+    appropriateCaseManagementDate: "appropriateCaseManagementDate",
+    appropriateCaseManagementNA: "appropriateCaseManagementNA",
+    initiatePublicHealthCounterMeasuresDate: "initiatePublicHealthCounterMeasuresDate",
+    initiatePublicHealthCounterMeasuresNA: "initiatePublicHealthCounterMeasuresNA",
+    initiateRiskCommunicationDate: "initiateRiskCommunicationDate",
+    initiateRiskCommunicationNA: "initiateRiskCommunicationNA",
+    establishCoordination: "establishCoordination",
+    responseNarrative: "responseNarrative",
+    incidentManagerName: "incidentManagerName",
+    notes: "notes",
+} as const;
+
+// TODO: Thinking for the future about generate this FormState by iterating over Object.Keys(diseaseOutbreakEvent)
+export function mapEntityToInitialFormState(
+    diseaseOutbreakEventWithOptions: DiseaseOutbreakEventWithOptions
+): FormState {
+    const { diseaseOutbreakEvent, options } = diseaseOutbreakEventWithOptions;
+    const {
+        teamMembers,
+        hazardTypes,
+        mainSyndromes,
+        suspectedDiseases,
+        notificationSources,
+        incidentStatus,
+    } = options;
+
+    const teamMemberOptions: UserOption[] = teamMembers.map(tm => ({
+        value: tm.username,
+        label: tm.name,
+        workPosition: tm.role?.name || "",
+        phone: tm.phone || "",
+        email: tm.email || "",
+        status: tm.status || "",
+        src: tm.photo?.toString(),
+        alt: tm.photo ? `Photo of ${tm.name}` : undefined,
+    }));
+
+    const hazardTypesOptions: PresentationOption[] = mapToPresentationOptions(hazardTypes);
+    const mainSyndromesOptions: PresentationOption[] = mapToPresentationOptions(mainSyndromes);
+    const suspectedDiseasesOptions: PresentationOption[] =
+        mapToPresentationOptions(suspectedDiseases);
+    const notificationSourcesOptions: PresentationOption[] =
+        mapToPresentationOptions(notificationSources);
+    const incidentStatusOptions: PresentationOption[] = mapToPresentationOptions(incidentStatus);
+
+    return {
+        id: diseaseOutbreakEvent?.id || "",
+        title: "Create Event",
+        saveButtonLabel: "Save & Continue",
+        isValid: false,
+        sections: [
```

Comment:
Huuuge array. Sometimes is nicer to move some object, so the main object is more readable. Something like: 

```
sections: [
   sections.main,
   sections.userInfo,
   ...
]
```